app/test_split_integer.py

```diff
@@ -2,20 +2,35 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 25
+    number_of_parts = 5
+    result = split_integer(value, number_of_parts)
+    assert sum(result) == value
```

Comment:
```suggestion
    assert sum(split_integer(25, 5)) == 25
```

I think  no need to create variable, if it is used only once